math/wxMaxima/Portfile

```diff
@@ -79,7 +79,9 @@ You might have to restart wxMaxima for this to take effect.
 wxMaxima and Maxima startup files can typically be found in:
   ~/.maxima
 
-ATTENTION: On macOS Catalina up to and including 10.15.3 there are issues with\
-the menus. (They don't work reliably.) Please use macOS 10.15.4 or later or\
-10.14.
+ATTENTION: On OSX Catalina up to including 10.15.X there are issues with the\
+menues (they don't work reliably). There is currently no solution for this.\
+wxMaxima on OSX up to including 10.14.X works fine.\
```

Comment:
More typos here.
Change:
`wxMaxima on OSX up to including`
to:
`wxMaxima on macOS up to and including`